app/controllers/workbooks_controller.rb

```diff
@@ -1,4 +1,6 @@
 class WorkbooksController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Instead of littering your controllers with this, you could create a new intermediate controller. 

```ruby 
class PrivateAreaController < ApplicationController
  before_action :authenticate_user!
end

class WorkbooksController < PrivateAreaController
end
```